components/script/dom/trustedtypepolicyfactory.rs

```diff
@@ -85,6 +86,49 @@ impl TrustedTypePolicyFactory {
         // Step 9: Return policy.
         Ok(policy)
     }
+
+    /// <https://w3c.github.io/trusted-types/dist/spec/#abstract-opdef-get-trusted-type-data-for-attribute>
+    #[allow(clippy::if_same_then_else)]
```

Comment:
I know this is ignoring a linter rule, but I want to closely follow the spec. It is already tricky to see what's what with the table. Especially as `TrustedScript` and `TrustedScriptURL` are nearly identical, it is easy to misinterpret the table from the spec and implement it wrongly. Therefore, while it indeed returns the same value, I think it is better to leave it as-is to assess correctness easier.